### PR TITLE
feat: enable search by block height

### PR DESCRIPTION
**Pull request by leahjlou**
Closes https://github.com/hirosystems/explorer/issues/895

Enables search by block height.

Screen Shot 2022-10-20 at 3 34 26 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Next let's make sure you have the tools to install the projects dependencies:
 
 We highly recommend using [Homebrew](https://brew.sh/).
 
-Now open your the Terminal, and make sure you are in the `/explorer` folder to run `yarn` to install the dependencies:
+Now open your Terminal, and make sure you are in the `/explorer` folder to run `yarn` to install the dependencies:
```
Comment (leahjlou) [inline README.md:24]: Also fixed small typo

File: src/features/search/use-search.ts [hunk 2]
```diff
@@ -1,26 +1,53 @@
...
-          return undefined;
+          if (data && 'found' in data) {
+            notFoundResult = data;
+          }
+        } catch (e) {}
+      }
+
+      try {
+        // Also check the block height
+        if (notFoundResult && isNumeric(id)) {
```
Comment (He1DAr) [inline src/features/search/use-search.ts:37]: maybe if it's numeric we skip `searchApi.searchById` and just fetch block by height?

**Review by He1DAr — APPROVED** [review]
LGTM, left 1 comment.

File: src/features/search/use-search.ts (hunk 2 above)
Comment (leahjlou) [reply src/features/search/use-search.ts:37]: Good point, if it's numeric we know for sure it's not a hash

**Pull request merged**